cards-api/Cards API.postman_collection.json

```diff
@@ -588,6 +589,35 @@
 						}
 					},
 					"response": []
+				},
+				{
+					"name": "8. Get a card transaction",
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "Authorization",
+								"value": "Bearer {{token}}",
+								"type": "text"
+							}
+						],
+						"url": {
+							"raw": "{{host}}/v3/spend/profiles/{{profile_id}}/cards/transactions/{{transaction-id}}",
```

Comment:
using `transaction_id` looks more consistent, do we need to add it to environment?

Reply:
I've reused `transaction-id` because it was already used by the Dispute API. The other Wise Platform APIs also mix dashes and underscore, which is why I'm not sure it's worth fixing but we could still do it for the Cards API Postman collection.

Reply:
hmm I did not notice it is already been used by dispute, I feel it still looks better to use `_` but I am fine with both :)